viewer/src/components/FactionWheel.vue

```diff
@@ -82,10 +88,7 @@ export default class FactionWheel extends Vue {
   }
 
   planetFill(planet: string) {
-    if (planet === Planet.Titanium || planet === Planet.Swamp) {
-      return "white";
-    }
-    return "black";
+    return planetFill(planet);
   }
```

Comment:
just a tip: you can do `planetFill = planetFill`

Reply:
I don't get it.